frontend/public/components/_horizontal-nav.scss

```diff
@@ -3,28 +3,31 @@ $co-m-horizontal-nav__menu-item-link-padding-lr: 15px;
 .co-m-horizontal-nav__menu {
   border-bottom: 1px solid $color-grey-background-border;
   display: flex;
-  white-space: nowrap;
   @media (min-width: $grid-float-breakpoint) {
     margin-left: (-$co-m-horizontal-nav__menu-item-link-padding-lr);
     padding: 0 30px;
   }
 }
 
-.co-m-horizontal-nav__menu-primary {
+.co-m-horizontal-nav__menu-primary,
+.co-m-horizontal-nav__menu-secondary {
   display: flex;
   list-style: none;
   margin: 0;
   padding: 0;
 }
 
 .co-m-horizontal-nav__menu-secondary {
-  display: flex;
-  list-style: none;
-  margin: 0 0 -1px; // -1px bottom so co-m-horizontal-nav__menu-item *::after are fully visible
+  flex-wrap: wrap;
   overflow-x: auto;
   overflow-y: hidden;
-  padding: 0;
   -webkit-overflow-scrolling: touch;
+  // hide tabs 4 - n on operator pages
+  .co-m-horizontal-nav__menu-item:nth-child(n + 4) {
```

Comment:
If I'm not mistaken, we still need to pass all pages to `<DetailsPage>` so the pages get created.  Otherwise there will be no list and details pages for each individual operand if there are more than one.

Reply:
It looks like the default details page is used as a fallback.
<img width="1623" alt="Screen Shot 2019-08-29 at 3 54 08 PM" src="https://user-images.githubusercontent.com/895728/63971772-5191ba00-ca75-11e9-8dee-bcad7b61a64f.png">

But no list page is created (which I navigated to via the breadcrumb on the details page).
<img width="1632" alt="Screen Shot 2019-08-29 at 3 54 14 PM" src="https://user-images.githubusercontent.com/895728/63971794-62dac680-ca75-11e9-8ded-c314b3650f9c.png">